extensions/shared/components/upgrade-nudge/index.jsx

```diff
@@ -62,7 +63,7 @@ export default compose( [
 		// The editor for CPTs has an `edit/` route fragment prefixed
 		const postTypeEditorRoutePrefix = [ 'page', 'post' ].includes( postType ) ? '' : 'edit';
 
-		const isWpcom = get( window, [ '_currentSiteType' ] ) === 'simple';
+		const isWpcom = isWpcomSite();
```

Comment:
In this case, we're actually more interested if we're in "Calypso or wp-admin", not if the site is .com site, but this works for now as it doesn't make any changes in flows.

Reply:
Should we say "simplewpcom", since we don't want to include Atomic in there?

Reply:
Agreed, let's call this `isSimpleSite`.

Reply:
Valid point!